configure.ac

```diff
@@ -7580,6 +7580,7 @@ PY_STDLIB_MOD([_testinternalcapi], [test "$TEST_MODULES" = yes])
 PY_STDLIB_MOD([_testbuffer], [test "$TEST_MODULES" = yes])
 PY_STDLIB_MOD([_testimportmultiple], [test "$TEST_MODULES" = yes], [test "$ac_cv_func_dlopen" = yes])
 PY_STDLIB_MOD([_testmultiphase], [test "$TEST_MODULES" = yes], [test "$ac_cv_func_dlopen" = yes])
+PY_STDLIB_MOD([_testsinglephase], [test "$TEST_MODULES" = yes], [test "$ac_cv_func_dlopen" = yes])
```

Comment:
Previously, `_testsinglephase` was enabled/disabled by piggy-backing on `_testmultiphase`. IMO, it is better to be explicit. Now, we get a clear indication from `./configure` and `make` if `_testsinglephase` is being built or not.